node-fs-readfile/cat.js

```diff
@@ -0,0 +1,18 @@
+import { readFile } from 'node:fs/promises';
+
+async function cat() {
+  try {
+    const fileName = process.argv.slice(2);
+
+    const fileTxt = await Promise.all(
+      fileName.map((fileName) => readFile(fileName, 'utf8'))
+    );
+
+    const concatTxt = fileTxt.join('\n');
+    console.log(concatTxt);
+  } catch (error) {
+    console.log(`Error: ${error.message}`);
```

Comment:
You usually want to log the entire error, not just the message field. That way you will get a stack trace that will help you figure out where things went wrong. (`console.log('Error:', error);`)